src/engraving/dom/instrtemplate.cpp

```diff
@@ -720,6 +722,16 @@ const InstrumentTemplate* searchTemplate(const String& name)
 const InstrumentTemplate* combinedTemplateSearch(const String& mxmlId, const String& name, const int transposition, int bank,
                                                  int program)
 {
+    struct InstrumentTemplateScored
+    {
+        const InstrumentTemplate* instrumentTemplate=nullptr;
+        int score = std::numeric_limits<int>::max();
+        InstrumentTemplateScored() {};
+    };
+
+    std::vector <InstrumentTemplateScored> instTemplateScoredVector;
+    int minLevenshteinDistance = std::numeric_limits<int>::max();
```

Comment:
Instead of building a scored list, and then at the end iterating through it to find the one with the lowest score, would it be possible to store only `int minLevenshteinDistance` and `InstrumentTemplate* templateWithMinLevenshteinDistance`?

Reply:
OK